includes/Admin/Settings.php

```diff
@@ -312,4 +340,70 @@ public function get_tabs() {
 		 */
 		return (array) apply_filters( 'wc_facebook_admin_settings_tabs', $tabs, $this );
 	}
+
+	/**
+	 * Add the Facebook for WooCommerce tabs to the Facebook Product Set taxonomy page.
+	 * Renders the tabs (hidden by default) at the stop of the page,
+	 * then moves them to the correct DOM location with JavaScript and displays them.
+	 *
+	 * @since 3.3.0
+	 */
+	public function add_tabs_to_product_sets_taxonomy() {
+		// Only load this on the edit-tags.php page
+		$screen                  = get_current_screen();
+		$is_taxonomy_list_page   = 'edit-tags' === $screen->base;
+		$is_taxonomy_term_page   = 'term' === $screen->base;
+		$is_taxonomy_page        = $is_taxonomy_list_page || $is_taxonomy_term_page;
+		$is_product_set_taxonomy = 'fb_product_set' === $screen->taxonomy && $is_taxonomy_page;
+
+		if ( $is_product_set_taxonomy ) {
+			$this->render_tabs( Settings_Screens\Product_Sets::ID );
+			?>
+				<style>
+					.facebook-for-woocommerce-tabs {
+						margin-top: 20px;
+						margin-bottom: 10px;
+						display: none;
+					}
+					#wpbody-content > .wrap > h1 {
+						margin-top: 65px;
+						font-size: 1.3em;
+						font-weight: 600;
+					}
+					<?php
+					// On the taxonomy list page in mobile view, account for the Screen Options tab.
+					if ( $is_taxonomy_list_page ) :
+						?>
+					@media (max-width: 782px) {
+						.facebook-for-woocommerce-tabs {
+								padding-top: 0;
+								position: relative;
+								top: -10px;
+								margin-bottom: -10px;
+							}
+						}
+					<?php endif; ?>
+				</style>
+				<script>
+						document.addEventListener('DOMContentLoaded', function() {
+								// Insert custom tabs into #wpbody
+								let tabs = document.querySelector('nav.facebook-for-woocommerce-tabs');
+								if (tabs) {
+										let wpbody = document.querySelector('#wpbody-content > .wrap');
+										if (wpbody) {		
+											// wpbody.querySelector('h1').remove();
```

Comment:
Nitpick: Looks like a leftover?

Reply:
Good spot, thanks. At least I got all the `console.log`s 😆 Removed in 31752c1f061a1a12b0f4aca21893b111d649dc43